as-vending/res/configuration.yaml

```
@@ -0,0 +1,43 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+Writable:
+  LogLevel: INFO
+
+Service:
+  Host: localhost
+  Port: 48099
+  StartupMsg: This microservice checks if ID numbers from REST requests are authenticated
+
+Clients:
+    core-command:
+      Protocol: "http"
+      Host: "localhost"
+      Port: 59882
+
+MessageBus:
+  Optional:
+    ClientId: "app-mqtt-export"
+
+# Using default Trigger config from common config
+ApplicationSettings:
```

Code defines the custom `Vending`  section.
```suggestion
Vending:
```